pkg/plugin/framework/restore_item_action_server.go

```diff
@@ -116,17 +116,20 @@ func (s *RestoreItemActionGRPCServer) Execute(ctx context.Context, req *proto.Re
 	// If the plugin implementation returned a nil updateItem (meaning no modifications), reset updatedItem to the
 	// original item.
 	var updatedItemJSON []byte
+	skipRestore := false
 	if executeOutput.UpdatedItem == nil {
 		updatedItemJSON = req.Item
 	} else {
 		updatedItemJSON, err = json.Marshal(executeOutput.UpdatedItem.UnstructuredContent())
+		skipRestore = executeOutput.SkipRestore
 		if err != nil {
 			return nil, newGRPCError(errors.WithStack(err))
 		}
 	}
 
 	res := &proto.RestoreItemActionExecuteResponse{
-		Item: updatedItemJSON,
+		Item:        updatedItemJSON,
+		SkipRestore: skipRestore,
```

Comment:
I think you want this to be `SkipRestore: executeOutput.SkipRestore`

Reply:
Yes, this is right. I'll change this.